modules/sno-du-enabling-workload-partitioning.adoc

```diff
@@ -1,16 +1,25 @@
 // Module included in the following assemblies:
 //
-// * scalability_and_performance/sno-du-enabling-workload-partitioning-on-single-node-openshift.adoc
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
 
 :_content-type: PROCEDURE
 [id="sno-du-enabling-workload-partitioning_{context}"]
 = Enabling workload partitioning
 
-Use the following procedure to enable workload partitioning for your single node deployments.
+A key feature to enable as part of single node installation is workload partitioning.
+This limits the cores allowed to run platform services maximizing the CPU core for
+application payloads. You must configure workload partitioning at cluster installation time.
+
+[NOTE]
+====
+Workload partitioning is required to be applied during installation.
```

Comment:
s/Workload partitioning is required to be applied during installation./Workload partitioning must be applied during installation.